cishouseholds/validate_class.py

```diff
@@ -108,23 +179,48 @@ def validate_udl(self, logic, error_message, columns):
                 "    - Falied to run check as required " + ",".join(missing) + " missing from dataframe"
             )  # functional
 
-    def execute_check(self, check, error_message, *params, subset=None, **kwargs):
+    def execute_check(self, check, error_message: object, *params, subset=None, **kwargs):
+        """
+        Validates check provided by a given instance of SparkValidate Class.
```

Comment:
mention whether it can be used on its own or as a hierarchy of other methods. 
On its own it would be a UDL (user defined logic)
mention as well that this is used for validate() and validate_column() to execute the checks.